benchmark/benchmark_roberta_model.py

```diff
@@ -0,0 +1,64 @@
+from argparse import ArgumentParser
+
+import torch
+import torchtext
+from benchmark.utils import Timer
+from torchtext.functional import to_tensor
+
+
+def basic_model_input(encoder):
+    transform = encoder.transform()
+    input_batch = ["Hello world", "How are you!"]
+    return to_tensor(transform(input_batch), padding_value=1)
+
+
+def _train(model, model_input):
+    model_out = model(model_input)
+    model_out.backward(torch.ones_like(model_out))
+    model.zero_grad()
+
+
+def run(args):
+    encoder = args.encoder
+    num_passes = args.num_passes
+    warmup_passes = args.num_passes
+    model_input = args.model_input
+
+    if encoder == "xlmr_base":
```

Comment:
Maybe use a dict and call `encoder = mapping_to_models[encoder]` to disentangle logic and correspondence between strings and models?